WalletWasabi/Wallets/CachedBlockProvider.cs

```diff
@@ -32,7 +33,8 @@ public async Task<Block> GetBlockAsync(uint256 hash, CancellationToken cancellat
 		if (block is null)
 		{
 			block = await BlockSourceProvider.GetBlockAsync(hash, cancellationToken).ConfigureAwait(false);
-			await BlockRepository.SaveAsync(block, cancellationToken).ConfigureAwait(false);
+			await LastSaveBlockTask;
```

Comment:
VS tells `.ConfigureAwait(false)` is missing from this line with a warning.

Should it be added?